src/Sulu/Bundle/ReferenceBundle/Tests/Functional/Infrastructure/Doctrine/Repository/ReferenceRepositoryTest.php

```diff
@@ -336,7 +330,7 @@ private function createReference(): ReferenceInterface
     /**
      * @return array<int, array<int, string>>
      */
-    public function filterProvider(): array
+    public static function filterProvider(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes

Reply:
This class does not in 2.5